payout-gateway.php

```diff
@@ -370,12 +370,33 @@ public function email_instructions( $order, $sent_to_admin, $plain_text = false
 		}
 
 
+		public function return_cents($amount) {
+		   $cents = intval($amount * 100);
+		   return $cents;
+		}
+
+
+
 
 
 		public function process_refund( $order_id, $amount = null, $reason = '' ) {
 
 
 
+			// Order object - to exclude some data
+			$order = wc_get_order( $order_id );
+			$payment_method_name = get_post_meta( $order_id, '_payment_method', true );
+
+			if ( ! $order ) {
+				return false;
+			}
+
+
+			if($payment_method_name != 'payout_gateway' ) {
+				return false;
```

Comment:
@SeducoDigital  nebolo by lepsie dat return true, neviem ako to funguje ale nie je to tak, ze ked vratis false tak ti zamietne akykolvek refund, resp. ti odpovie, ze to nepreslo. Ci?